usb.go

```diff
@@ -31,21 +31,31 @@ func initUsbGadget() {
 		}
 	})
 
+	gadget.SetOnKeysDownChange(func(state usbgadget.KeysDownState) {
+		if currentSession != nil {
+			writeJSONRPCEvent("keysDownState", state, currentSession)
+		}
+	})
+
 	// open the keyboard hid file to listen for keyboard events
 	if err := gadget.OpenKeyboardHidFile(); err != nil {
 		usbLogger.Error().Err(err).Msg("failed to open keyboard hid file")
 	}
 }
 
-func rpcKeyboardReport(modifier uint8, keys []uint8) error {
+func rpcKeyboardReport(modifier byte, keys []byte) (usbgadget.KeysDownState, error) {
```

Comment:
We now return the KeysDownState from the keyboardReport, which will be ignored by old browser clients.